chroot-bin/croutonxinitrc-wrapper

```diff
@@ -78,7 +78,42 @@ if [ "$XMETHOD" = 'xephyr' ]; then
     croutonwheel $CROUTON_WHEEL_PARAMS &
 fi
 
-# xbindkeys and other input-related apps are not needed for kiwi
+# Launch system-wide triggerhappy if it isn't running
+mkdir -m 775 -p /tmp/crouton-lock
+{
+    # Only let one instance *really* run at a time
+    flock 3
+    cmdprev='/usr/local/bin/croutoncycle p'
+    cmdnext='/usr/local/bin/croutoncycle n'
+    echo "#
+# Ctrl+Alt+Shift+F1 = previous. Require full key release to avoid stuck keys.
+KEY_F1+KEY_LEFTSHIFT+KEY_LEFTCTRL+KEY_LEFTALT@ 1 @p
+KEY_F1@p 0 $cmdprev
+KEY_LEFTSHIFT@p 0 $cmdprev
+KEY_LEFTCTRL@p 0 $cmdprev
+KEY_LEFTALT@p 0 $cmdprev
+KEY_F1@p 0 @
+KEY_LEFTSHIFT@p 0 @
+KEY_LEFTCTRL@p 0 @
+KEY_LEFTALT@p 0 @
+#
+# Ctrl+Alt+Shift+F2 = next Require full key release to avoid stuck keys.
+KEY_F2+KEY_LEFTSHIFT+KEY_LEFTCTRL+KEY_LEFTALT@ 1 @n
+KEY_F2@n 0 $cmdnext
+KEY_LEFTSHIFT@n 0 $cmdnext
+KEY_LEFTCTRL@n 0 $cmdnext
+KEY_LEFTALT@n 0 $cmdnext
+KEY_F2@n 0 @
+KEY_LEFTSHIFT@n 0 @
+KEY_LEFTCTRL@n 0 @
+KEY_LEFTALT@n 0 @
+#" | /usr/sbin/thd --triggers /dev/stdin \
```

Comment:
Anyway, expanded that to monitor all event files (and watch for new ones), so we're good to go.
Side benefit: this can probably be used to monitor for things like lid switch events.